lib/winapi/debug.c

```diff
@@ -0,0 +1,17 @@
+#include <winbase.h>
+#include <winnt.h>
+
+#include <string.h>
+
+void WINAPI OutputDebugStringA (LPCTSTR lpOutputString)
```

Comment:
MS docs describe that this function calls `DbgPrint()` internally. I couldn't spot any difference in behavior when using this functions vs. using DbgPrint (when called on Retail or Debug BIOSes), so is this something we should do, too?

Reply:
MS docs are either wrong (changes over time?), or `DbgPrint` was inlined (which I doubt).

My version is based on reverse engineering of games, and it does not call any function which does string formatting (which `DbgPrint` would do).

Another difference is that this one doesn't need a function call and can be LTCG'd; it also isn't imported from the kernel, which can be relevant in some cases.